### DISCUSSION TITLE
Make an enterprise version that allows selection of private repositories and limit software that can be installed

### DISCUSSION THREAD

**Discussion by nclinch** [Devolutions/UniGetUI#1330] [General]
Make an enterprise version that allows selection of private repositories:
I would love to have this for my end users but I want to limit what they can install.
I want an option for new versions of software to be automatically installed or not.

I believe the chocolatey recommends private repositories.

**Comment by marticliment** [comment]
You can set up winget or chocolatey to use private repositories. WingetUI will adapt to that.

Then, you can disable automatic updates and serve WingetUI updates from that same private repository
